docs/cub/tuning.rst

```diff
@@ -247,18 +247,53 @@ that we don't intend to provide separate tuning policies for.
 Search Process
 --------------------------------------------------------------------------------
 
-To get started with tuning / benchmarking, you need to configure CMake. You can use the following command:
+To get started with tuning, you need to configure CMake.
+You can use the following command:
 
 .. code:: bash
 
   $ mkdir build
   $ cd build
   $ cmake .. --preset=cub-tune -DCMAKE_CUDA_ARCHITECTURES=90 # TODO: Set your GPU architecture
```

Comment:
Is this actually working? because last time I tried the cmake needed to be executed in the top level folder where the preset is. We can nevertheless simply clean this up
```suggestion
  $ cmake --preset=cub-tune -DCMAKE_CUDA_ARCHITECTURES=90 # TODO: Set your GPU architecture
```

This would require us to adopt the folder structure below

Reply:
Yes this command works fine, just tested on my workstation:
```
bgruber@concorde:~/dev/cccl$ mkdir build_tt
bgruber@concorde:~/dev/cccl$ cd build_tt
bgruber@concorde:~/dev/cccl/build_tt$ cmake .. --preset=cub-tune -DCMAKE_CUDA_ARCHITECTURES=86
Preset CMake variables:

  CCCL_ENABLE_BENCHMARKS:BOOL="TRUE"
  CCCL_ENABLE_C:BOOL="FALSE"
  CCCL_ENABLE_CUB:BOOL="TRUE"
  CCCL_ENABLE_CUDAX:BOOL="FALSE"
  CCCL_ENABLE_EXAMPLES:BOOL="FALSE"
  CCCL_ENABLE_LIBCUDACXX:BOOL="FALSE"
  CCCL_ENABLE_TESTING:BOOL="FALSE"
  CCCL_ENABLE_THRUST:BOOL="FALSE"
  CMAKE_BUILD_TYPE="Release"
  CUB_ENABLE_EXAMPLES:BOOL="FALSE"
  CUB_ENABLE_HEADER_TESTING:BOOL="FALSE"
  CUB_ENABLE_TESTING:BOOL="FALSE"
  CUB_ENABLE_TUNING:BOOL="TRUE"

-- The CXX compiler identification is GNU 13.2.0
...
```